apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -315,7 +321,10 @@ class TopInstructions extends Component {
       this.state.feedbacks.length > 0 &&
       (this.state.feedbacks[0].comment || this.state.feedbacks[0].performance);
     const displayFeedbackTeacher =
-      this.props.viewAs === ViewType.Teacher && this.state.rubric;
+      this.state.displayFeedbackTeacherFacing ||
+      (!this.state.displayFeedbackTeacherFacing &&
```

Comment:
why would `this.state.displayFeedbackTeacherFacing` be false here (i.e., for teachers in the experiment)?

Reply:
so we show the feedback tab for teachers in the experiment in two cases:
1) viewing student work
2) viewing level normally as a teacher(not viewing student work) and there is a rubric

So the biggest reason it is false is if you are not viewing student work and there is no rubric

Reply:
okay, that makes sense. i think some of this complexity will go away once there is no longer an experiment to deal with, but the display logic is getting pretty complex here. we should revisit when we're removing the experiment, or it may just fix itself 😄